toolkit/tools/imagegen/configuration/user_test.go

```diff
@@ -71,15 +71,15 @@ func TestShouldFailParsingInvalidUIDLowerBound_User(t *testing.T) {
 	testUser.UID = "-2"
 	err := testUser.UIDIsValid()
 	assert.Error(t, err)
-	assert.Equal(t, "invalid value for UID (-2)", err.Error())
+	assert.Equal(t, "invalid value for UID (-2), not within [0, 60000]", err.Error())
```

Comment:
Similar comment: Here and elsewhere I'd also avoid using a hard-coded string ("-2" in this case) and use the value of `testUser.UID` or other variable we're using for the tests instead.

Reply:
I would argue in the test we want the hard coded value to make sure it is exactly as we expect.